src/frontend/js/components/CourseRunEnrollment/index.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { useCallback, useEffect, useReducer } from 'react';
+import { useCallback, useEffect, useReducer } from 'react';
+import * as React from 'react';
```

Comment:
This seems worse than before: 2 lines and a wildcard import. Can we just leave it like it was?

Reply:
Oh yes ! I trusted `react-codemod`...

Reply:
IMO we can leave `import React from 'react'` wherever it inserted `import * as React`. What do you think?

Reply:
Yes I also find the syntax better. But I took the opportunity to import only stuff used when there is only one thing used from React.

Reply:
No more React Football Club!